sdk/servicebus/Azure.Messaging.ServiceBus/src/Azure.Messaging.ServiceBus.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="Azure.Core.Experimental" />
```

Comment:
What gates to we have in place to ensure servicebus doesn't ship with this dependency?

Reply:
You mean GA? Nothing automatic, just our vigilance.

Reply:
:(  That scares me a little. I cannot remember but nuget might complain if you are building a GA package with a dependency on a prelease package.